src/main/resources/site/parts/omStatistikken/omStatistikken.ts

```diff
@@ -261,16 +268,33 @@ function getAccordion(
 }
 
 function getItems(category: Category, variables: Array<string>, phrases: Phrases): Array<AccordionItem> {
-  return variables.map((variable) => {
-    return {
-      title: phrases[variable],
-      body: category[variable]
-        ? processHtml({
-            value: category[variable].replace(/&nbsp;/g, ' '),
-          })
-        : phrases.notRelevant,
+  const items: Array<AccordionItem> = []
+  variables.forEach((variable) => {
+    //TODO: When all om statistikken has content in new items remove check isNewItem.
```

Comment:
I don't quite get what this TODO means. 
Is there any way we can make sure we actually remove the isNewItem check in the future?

If it can be done now, please do. 
If it has to be done later, please explain more in the comment exactly what has to be the case in order to remove the check. So that a developer can confidently remove the check in the future 😄

Reply:
Har endret litt på teksten i TODO